packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -75,6 +75,14 @@ class TestCommand extends FlutterCommand {
         help: 'Whether matchesGoldenFile() calls within your test methods should '
               'update the golden files rather than test for an existing match.',
       )
+      ..addFlag('watch',
```

Comment:
Even though there is some file watching going on, I don't think that is the defining feature of this. Maybe "interactive?" I'm not quite sure.

Reply:
I'm open to either, I have it hidden so that we can feel free to change this in the near future.

Reply:
Leaving as is until someone has a better name for the flag.  Since the files are in fact being watched, I think the current name makes sense.